msg/Strings.msg

```diff
@@ -0,0 +1,13 @@
+string string_value
+string string_value_default0 "Hello world!"
```

Comment:
Maybe start indexing the field name with 1?

Same below.

Reply:
Sure, b311904